### DISCUSSION TITLE
UI Changes in MAUI of .NET 8

### DISCUSSION THREAD

**Discussion by MAUIoxo** [dotnet/maui#17423] [General]
I tried Visual Studio 2022 17.8  8.0.100-preview.7.23376.3 and also installed .NET 8   8.0.100-rc.1.23455.8 and deployed my App which is currently in development.

What I could see is that there seemed to be quite some changes in UI and what was obvious directly was that the standard fonts looked  differently and not as smooth as they used to look. Also other elements did not look as smooth as before and e.g. Switches assumed to be bigger.

Therefore my question, is it planned that they change back to the smoother way they looked with .NET 7 from a RC to the version that is finally released or do I have to assume they stay as I can see them right now? Also the question is if I can assume there will be a lot of changes between a RC and the finally released one? 

I am asking this because I expect a lot of bug fixes in .NET MAUI also in parts I have in my .NET 7 App right now and therefore I will most likely switch to .NET 8 when it is released. So, question is if I better should wait to adapt my App or better wait to the final release?

**Comment by maonaoda** [comment]
The best option is to give up maui.
